OAuth1Migration/Fitbit.OAuth1Migration/FitbitClientOA1Factory.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Net.Http;
+using Fitbit.Api.Portable;
+
+namespace Fitbit.OAuth1Migration
+{
+    public static class FitbitClientOA1Factory
```

Comment:
Not a big fan of the name, but as it will be relatively short lived I'll survive ;-)